CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 0.32.1 / 2018-02-22
```

Comment:
I'd say the changes are minor with the inclusion of the hidden text rather than just a bug fix. Can this version become `0.33.0` please? `package.json` will need to be updated too.